_posts/2019-06-11-sgid-then-and-now.md

```diff
@@ -0,0 +1,128 @@
+
+![1999 SGID User Guide Binder]({{ "/images/SGID_1999_binders_cover_sm.png" | prepend: site.baseurl }}){: .flex .flex--center}
+
+- **SGID meets the World Wide Web (the early 2000s)**
+  - By the early 2000s the SGID user's guide, data, and data viewer were all solely available over the Web. AGRC was using ArcSDE 8.2 to maintain the data and ArcIMS for the interactive mapping website. Users could also use a separate ArcIMS site to create their own exportable map using a [custom map authoring tool]({{ "/images/SGID_early2000s_create_custom_map.gif" | prepend: site.baseurl }}).
+
+[![SGID Early 2000s]({{ "/images/SGID_early2000s_sm.png" | prepend: site.baseurl }}){: .flex .flex--center}]({{ "/images/SGID_early2000s.png" | prepend: site.baseurl }})
+
+### `Where is the SGID Now?`
+{: .text-left}
+
+Technology and the use of GIS have come a long way since the inaugural SGID of 1991.   What hasn't changed, though, is the notion of a centralized repository for searching and fetching statewide GIS data.  Sure, file formats and how we access data has changed, but the need for open, readily-accessible GIS data remains the same.
+
+The current makeup of the SGID is distributed, by design.  Internally, much of the framework data is managed on a SQL ArcSDE Database.  From there, data layers are distributed accordingly.
+
+The [index catalog]({{ "/data/sgid-index/" | prepend: site.baseurl }}) and user guide are located on the [AGRC’s website]({{ "/data/" | prepend: site.baseurl }}).  Here you will find descriptions of the data, update dates, data downloads, as well as links to other places where you can access the data.  It is recommended that you begin your data search there.
+
+![SGID 2019 User Guide]({{ "/images/SGID_2019_webpage_data.png" | prepend: site.baseurl }}){: .flex .flex--center}
+
+In order to meet the diverse needs of our users and applications, SGID data is served on a variety of platforms in a variety of formats, including direct database access, downloadable data, and API web services.
+
+Here are some current SGID stats, showing top accessed data layers by platform:
+
+- **[AGRC Website](https://gis.utah.gov/data)** - Page-views from the past 12 months - rounded to the nearest one-hundred
+  - Parcels - (30,900 page-views)
+  - Elevation and Terrain - (20,400)
+  - Aerial Photography - (20,100)
+  - Transportation - (18,200)
+  - USGS Scanned Topo Maps - (15,900)
+  - Boundaries - Landownership - (15,600)
+  - Boundaries - City, County, and State - (15,300)
+  - Oil and Gas - (8,000)
+  - Water - (7,600)
+  - Basemap and Imagery - (5,300)
+
+- **[ArcGIS Online](https://utah.maps.arcgis.com)** - The most popular content from June 2013 to May 2019, rounded to the nearest one-thousand
+  - SBI Operations (which includes: Law Enforcement Boundaries, City Courts, County Courts, State District Courts, UHP Dispatch Zones) - (67,000 views)
+  -  Roads (minor and major roads combined stats) - (64,000)
```

Comment:
should this one say the measurement? requests?

Reply:
i took the term "most popular content" from the agol dashboard where I got the numbers.  It's a little vague to me as well, but I better stick to it b/c I don't know the exact metric.